.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -52,6 +52,7 @@ body:
         - quirc
         - sh2lib
         - spi_nand_flash
+        - soft_uart
```

Comment:
I slightly don't agree with the component name. In fact, this driver is not a bit-bang software driver, it still utilized the hardware's transceiver. 

Proposed another name `uart_emu`. Then we can add more emulator backend drivers in the future (e.g. dedicated GPIO).